plugins/apel/.gitignore

```diff
@@ -9,4 +9,4 @@ __pycache__/
 dist/
 build/
 _version.py
-*.local
\ No newline at end of file
+*.local.*
```

Comment:
Is this change intended? This is not really connected to the CI actions.

Reply:
Yes, I just wanted to squeeze that in ;)